workflows/iuf/operations/management-nodes-rollout/management-worker-nodes-reboot.yaml

```
@@ -0,0 +1,547 @@
+            - reboot-node
+          templateRef:
+            name: ncn-reboot-common
+            template: wait-for-cloud-init
+          arguments:
+            parameters:
+              - name: dryRun
+                value: "{{$.DryRun}}"
+              - name: targetNcn
+                value: "{{inputs.parameters.targetNcn}}"
+              - name: mediaHost
+                value: 'ncn-m001'
+        - name: wait-for-k8s
+          dependencies: 
+            - wait-for-cloud-init
+          templateRef:
+            name: kubectl-and-curl-template
+            template: shell-script
+          arguments:
+            parameters:
+              - name: dryRun
+                value: "{{$.DryRun}}"
+              - name: mediaHost
+                value: 'ncn-m001'
+              - name: scriptContent
+                value: |
+                  TARGET_NCN="{{inputs.parameters.targetNcn}}"
+                  # Wait for Kubernetes membership
+                  echo "INFO Waiting for node to rejoin Kubernetes: ${TARGET_NCN}"
+                  MAX_RETRY=45
+                  n=0
+                  until csi automate ncn kubernetes --action is-member --ncn "$TARGET_NCN" --kubeconfig mykubeconfig/admin.conf; do
+                    n=$((n+1))
+                    if [[ $n -ge $MAX_RETRY ]]; then
+                      echo "ERROR: Node did not rejoin Kubernetes in time."
+                      exit 1
+                    fi
+                    echo "DEBUG Waiting for node to rejoin Kubernetes: ${TARGET_NCN}"
+                    sleep 15
+                  done
+        - name: wait-for-cfs
+          dependencies: 
+            - wait-for-cloud-init
+          templateRef:
+            name: ncn-reboot-common
+            template: wait-for-cfs
+          arguments:
+            parameters:
+              - name: dryRun
+                value: "{{$.DryRun}}"
+              - name: targetNcn
+                value: "{{inputs.parameters.targetNcn}}"
+        - name: uncordon-node
+          dependencies: 
+            - wait-for-k8s
+          templateRef:
+            name: ssh-template
+            template: shell-script
+          arguments:
+            parameters:
+              - name: dryRun
+                value: "{{$.DryRun}}"
+              - name: mediaHost
+                value: 'ncn-m001'
+              - name: scriptContent
+                value: |
+                  TARGET_NCN="{{inputs.parameters.targetNcn}}"
+                  # Uncordon the node
+                  kubectl uncordon "$TARGET_NCN"
+                  echo "INFO Successfully Uncordon: ${TARGET_NCN}"
+    - name: move-to-next-set
+      inputs:
+        parameters:
+          - name: count
+      script:
+        image: artifactory.algol60.net/csm-docker/stable/docker.io/alpine/git:2.32.0
+        command: [sh]
+        source: |
+          #!/bin/sh
+          current_count="{{inputs.parameters.count}}"
+          counter=$(($current_count + 1 ))
```

`$` is not needed inside `$((...))`
```suggestion
          counter=$((current_count + 1 ))
```